skipper.yaml

```diff
@@ -0,0 +1,16 @@
+registry: quay.io
+build-container-image: node-feature-discovery-operator-build
+
+containers:
+  node-feature-discovery-operator-build: Dockerfile.node-feature-discovery-operator-build
+volumes:
+    - $HOME/.cache/go-build:$HOME/.cache/go-build
+    - $HOME/go/pkg:/go/pkg
+    - $HOME/.cache/golangci-lint:$HOME/.cache/golangci-lint
+    - $HOME/.docker:$HOME/.docker
+    - $HOME/.kube/:$HOME/.kube
+    - $HOME/.minikube/:$HOME/.minikube
```

Comment:
Are these needed for something? Just thinking the principle of least privileges/smallest blast radius

Reply:
i will remove kube and minikube mappings.If i am not mistaken, then docker mappings are needed